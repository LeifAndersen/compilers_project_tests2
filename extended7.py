def a() -> "bar":
 pass

def a(x: "bar",):
 return x

def c(x: "bar", y: "baz") -> "bleb":
 return x*y


