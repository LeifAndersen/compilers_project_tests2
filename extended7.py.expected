(program
 (def (a) (suite (pass)))
 (def (a x) (suite (return x)))
 (def (c x y) (suite (return (term x ("*" y))))))

